tests/foreman/ui_airgun/conftest.py

```diff
@@ -117,7 +146,12 @@ def test_foo(session):
                 session.architecture.create({'name': 'bar'})
 
     """
-    return Session(test_name, module_user.login, module_user.password)
+    session = Session(test_name, module_user.login, module_user.password)
+    _signal_session_init()
```

Comment:
I'd also start that timeout only in case we're working with saucelabs - we're facing the issue with loosing saucelabs connection, local runs are not affected, so we shouldn't forbid long local UI sessions.

Reply:
do not think that we have tests that work fot 7000 seconds

Reply:
7000 secs was added to wrap up a test with 2 hrs in CI , I had seen some capsule tests taking more than a hour, so this was a approx estimate. We can look into reducing this
@lpramuk @rplevka any inputs here